lib/web/users.go

```diff
@@ -305,9 +311,10 @@ type userTraits struct {
 // - if the value is an empty array we remove every element from the trait
 // - otherwise, we replace the list for that trait
 type saveUserRequest struct {
-	Name   string     `json:"name"`
-	Roles  []string   `json:"roles"`
-	Traits userTraits `json:"traits"`
+	Name      string              `json:"name"`
+	Roles     []string            `json:"roles"`
+	Traits    userTraits          `json:"traits"`
+	AllTraits map[string][]string `json:"allTraits"`
```

Comment:
Nit: What do you think about naming this `OverrideTraits` or `SetTraits`? To make it more clear that if it's provided, it will set all user traits?

Also, can you add godocs on these fields.

Reply:
tl;dr: I updated `Traits` to `TraitsPreset` and added Godoc for clarity. Json tags are as is since they are also used in other parts of frontend. Lmk if you still find it confusing. 

The names `Traits` and `AllTraits` predates this PR and it isn't super straightforward to update these. I took a second tour today but there are other parts in frontend codebase (I found one discovery flow and not 100% on access list) that make use of these fields.